mvpa/cosmo_target_dsm_corr_measure.m

```diff
@@ -18,8 +18,10 @@
 %                  pair-wise distances between samples in ds. It accepts
 %                  any metric supported by pdist (default: 'correlation')
 %     .type        (optional) type of correlation between target_dsm and
-%                  ds, one of 'Pearson' (default), 'Spearman', or
-%                  'Kendall'.
+%                  ds, one of 'Pearson' (default) or 'Spearman'. If the
+%                  'regress_dsm' option is used, then the specified
+%                  correlation type is used for the partial correlaton
+%                  computation.
```

Comment:
Disabling Kendall breaks backwards compatibility, no? Kendall should still be allowed when not using regress_dsm